changelogs/fragments/3951-scaleway_compute_add_project_id.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - scaleway_compute - add poosibility to use project identifier (new ``project`` option) instead of deprecated organization identifier (https://github.com/ansible-collections/community.general/pull/3951).
```

Comment:
```suggestion
  - scaleway_compute - add posibility to use project identifier (new ``project`` option) instead of deprecated organization identifier (https://github.com/ansible-collections/community.general/pull/3951).
```